src/main/java/nopol_examples/nopol_example_4/NopolExample.java

```diff
@@ -11,6 +11,7 @@ public boolean canBeDividedby3(String a) {
 		int initializedVariableShouldBeCollected = 2;
 		int uninitializedVariableShouldNotBeCollected;
 		int otherInitializedVariableShouldBeCollected;
+                int foo = 1;
 		
 		if ( a.length() == 0 ){
 			return false;
```

Comment:
This line (17) has been identified with a suspiciousness value of 0.00%.
The following failing tests covered this line: {}